tests/certification/workflows/temporal/temporal_test.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package temporal_test
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+	"time"
+	// "github.com/dapr/components-contrib/workflows"
+)
+
+func TestTemporal(t *testing.T) {
+	log := logger.NewLogger("dapr.components")
+	ports, err := dapr_testing.GetFreePorts(2)
+	assert.NoError(t, err)
+
+	currentGrpcPort := ports[0]
+	currentHTTPPort := ports[1]
+}
```

Comment:
Does this file need to be removed? My previous comment was marked as resolved, and I was expecting this would have been removed. I'm not sure what the current expectation is. We don't appear to be actually testing anything here.